README.md

```diff
@@ -226,12 +226,43 @@ class SimpleLitAPI(LitAPI):
 
 if __name__ == "__main__":
     api = SimpleLitAPI()
-    server = LitServer(api, accelerator="cpu")
+    server = LitServer(api, accelerator="auto")
     server.run(port=8888)
 ```
 
 </details>
 
+<details>
+    <summary>Automatic accelerator</summary>
+&nbsp;
+
+LitServe can automatically select the appropriate GPU or CPU, whichever is available on the user's machine.
+
+```python
+# Automatically selects the available accelerator
+
+api = SimpleLitAPI()
+
+# these two are equivalent
+server = LitServer(api)
+server = LitServer(api, accelerator='auto')
```

Comment:
@lantiga @aniketmaurya 
this is how we should show things that happen automatically... an empty () doesn't show what this does correctly.